src/DataTable.php

```diff
@@ -778,7 +852,7 @@ public function loadRelations(): array
             })
             ->map(function (Relation $relation) use ($basis) {
                 return [
-                    'value' => $relation->name,
+                    'value' => Str::snake($relation->name),
```

Comment:
Isn't this a gamble? I mean, a relation name does not necessarily mean that the table is named the same only in snake_case.

Reply:
when doing a ->toArray() all relation keys are snake case, no matter if they are camel or snake in the class.